modules/campaignion_context/campaignion_context.info

```diff
@@ -7,6 +7,8 @@ dependencies[] = cck_blocks
 dependencies[] = context
 dependencies[] = ctools
 dependencies[] = recent_supporters
+dependencies[] = share_light
+dependencies[] = views
```

Comment:
What about (some form of) pgbar?

Reply:
I haven’t added these manually. I wonder why pgbar wasn’t pulled in if we have a related context reaction.

Reply:
I’ve added it manually. Still strange that features doesn’t find this automatically :shrug: I think it’s missing a proper support for cck_blocks.